QMLComponents/jasptheme.h

```diff
@@ -714,7 +725,8 @@ private slots:
 						_fontRCode,
 						_fontCode,
 						_fontGroupTitle,
-						_fontPrefOptionsGroupTitle;
+						_fontPrefOptionsGroupTitle,
+						_fontALTNavTag;
```

Comment:
Why add an extra font for this?

Not saying it is a bad idea, just curious why

Reply:
The others were a bit large. The size of the tag is determined by the font size of the text